packages/keyv/src/index.d.ts

```diff
@@ -85,7 +85,12 @@ declare namespace Keyv {
 		/** Specify an adapter to use. e.g `'redis'` or `'mongodb'`. */
 		adapter?: 'redis' | 'mongodb' | 'mongo' | 'sqlite' | 'postgresql' | 'postgres' | 'mysql' | undefined;
 		/** Enable compression option **/
-		compress?: Record<string, unknown> | undefined;
+		compression?: CompressionAdapter | undefined;
+	}
+
+	interface CompressionAdapter {
+		compress: (value: any) => any;
+		decompress: (value: any) => any;
```

Comment:
can we add in `options` parameter?

Reply:
@jaredwray Earlier we are supporting `options` parameter but you said pass the `options` parameter in constructor

Reply:
I would like it to be passed via a property or as a parameter in the constructor. Either way it should work and we should test for that.

Reply:
ok @jaredwray I will do that